src/main/java/mate/model/Car.java

```diff
@@ -0,0 +1,31 @@
+package mate.model;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Car {
+    private Long id;
+    private String model;
+    private Manufacturer manufacturer;
+    private List<Driver> drivers;
+
+    public Car() {
+    }
+
+    public Car(String model, Manufacturer manufacturer) {
+        this.model = model;
+        this.manufacturer = manufacturer;
+        drivers = new ArrayList<>();
```

Comment:
Why we create new List every time when create Car?

Reply:
![Untitled](https://user-images.githubusercontent.com/59320919/104235987-787c4080-545e-11eb-8cc6-bceb343b25a1.png)

Reply:
![Untitled](https://user-images.githubusercontent.com/59320919/104236289-f7717900-545e-11eb-8571-19d68f658f6e.png)